qa-ibd.planx-pla.net/portal/gitops.json

```diff
@@ -424,6 +430,37 @@
       "dropdowns": {}
     }
   ],
+  "studyViewerConfig": [
+    {
+      "dataType": "project",
+      "title": "Projects",
```

Comment:
oh, my bad, we might need to call it "Studies" instead of "Projects" if that's the terminology used in IBD. but i think for now we're good, Phil can let us know what he prefers